src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceServiceSpec.scala

```diff
@@ -1,250 +1,339 @@
 package org.broadinstitute.dsde.firecloud.service
 
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
 import org.broadinstitute.dsde.firecloud.mock.MockUtils._
-import org.broadinstitute.dsde.firecloud.mock.MockWorkspaceServer
-import org.broadinstitute.dsde.firecloud.mock.MockTSVFormData
-import org.broadinstitute.dsde.firecloud.model.{MethodConfiguration, WorkspaceEntity, WorkspaceName}
-import spray.http.StatusCodes._
-
-import spray.httpx.SprayJsonSupport._
-import spray.json.DefaultJsonProtocol._
+import org.broadinstitute.dsde.firecloud.mock.{MockUtils, MockTSVFormData}
+import org.broadinstitute.dsde.firecloud.model._
 import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol._
 
-class WorkspaceServiceSpec extends ServiceSpec with WorkspaceService {
+import org.mockserver.integration.ClientAndServer
+import org.mockserver.integration.ClientAndServer._
+import org.mockserver.model.HttpRequest._
+import org.scalatest.BeforeAndAfterEach
+import spray.http.HttpMethods
 
-  import org.broadinstitute.dsde.firecloud.model.ErrorReport.errorReportRejectionHandler
+import spray.http.StatusCodes._
+import spray.json._
 
-  def actorRefFactory = system
+class WorkspaceServiceSpec extends ServiceSpec with WorkspaceService with BeforeAndAfterEach {
 
-  private final val ApiPrefix = "/workspaces"
+  def actorRefFactory = system
 
-  override def beforeAll(): Unit = {
-    MockWorkspaceServer.startWorkspaceServer()
+  val workspace = WorkspaceEntity(
+    Some("namespace"),
+    Some("name")
+  )
+
+  // Mock remote endpoints
+  private final val workspacesRoot = "workspaces"
+  private final val workspacesPath = workspacesRoot + "/%s/%s".format(workspace.namespace.get, workspace.name.get)
+  private final val methodconfigsPath = workspacesRoot + "/%s/%s/methodconfigs".format(workspace.namespace.get, workspace.name.get)
+  private final val updateAttributesPath = workspacesRoot + "/%s/%s/updateAttributes".format(workspace.namespace.get, workspace.name.get)
+  private final val aclPath = workspacesRoot + "/%s/%s/acl".format(workspace.namespace.get, workspace.name.get)
+  private final val clonePath = workspacesRoot + "/%s/%s/clone".format(workspace.namespace.get, workspace.name.get)
+  private final val lockPath = workspacesRoot + "/%s/%s/lock".format(workspace.namespace.get, workspace.name.get)
+  private final val unlockPath = workspacesRoot + "/%s/%s/unlock".format(workspace.namespace.get, workspace.name.get)
+
+  private final val workspaceBasePath = FireCloudConfig.Rawls.authPrefix + FireCloudConfig.Rawls.workspacesPath
+  private final val tsvImportPath = "/" + workspacesRoot + "/%s/%s/importEntities".format(workspace.namespace.get, workspace.name.get)
```

Comment:
Ah - I do know the difference. 
The urls without "/" are all mock urls from calls to Rawls. 
All of the urls with the "/" (or just the tsvImportPath path) is a local url and is what's used for calls to Orchestration. 
From a code organization point of view, this could be better laid out.